app/views/dashboard/_sidebar.html.erb

```diff
@@ -1,5 +1,57 @@
-<br />
+  <br />
+ 	<div class="panel panel-default">
+	  <div class="panel-heading">
+	    <h1 class="panel-title">Ask a question</h1>
+	  </div>
+	  <div class="panel-body">
+	    <p>The easiest way to start with Public Lab community is to ask a question.<br />See questions and ask your own here:</p>
+	    <a class="btn btn-primary btn-sm" href="/questions">Questions &raquo;</a> 
+	  </div>
+	</div>
+  <ul class="nav nav-tabs">
+    <li class="active">
+      <a class="wikis-tab" href="#wiki" data-toggle="tab">
+        <span class="visible-xs"><i class="fa fa-book"></i></span>
+        <span class="hidden-xs"><%= t('dashboard._sidebar.wiki') %></span>
+      </a>
+    </li>
+    <li>
+      <a class="lists-tab" href="#lists" data-toggle="tab">
+        <span class="visible-xs"><i class="fa fa-comment"></i></span>
+        <span class="hidden-xs"><%= t('dashboard._sidebar.discussion_list') %></span>
+      </a>
+    </li>
+  </ul>
 
-<div id="wiki">
-  <%= render partial: "dashboard/wiki", locals: { wikis: @wikis } %>
-</div>
+  <div class="tab-content">
+
+    <div class="tab-pane active" id="wiki">
+
+      <%= render partial: "dashboard/wiki", locals: { wikis: @wikis } %>
+
+    </div>
+
+    <div class="tab-pane" id="lists">
```

Comment:
Do we need the lists content here? I think it was removed in a previous project. Be sure to update your master branch before adding new work to it, or older changes will creep back into your code like this! 

For now, compare to the latest publiclab/plots2 master branch's file to see what's been removed, and let's update this to match, sound good?